spec/javascripts/accordion-with-descriptions-spec.js

```diff
@@ -2,215 +2,242 @@ describe('An accordion with descriptions module', function () {
   "use strict";
 
   var $element;
-
-  beforeEach(function() {
-    $element = $('<div class="subsections js-hidden" data-module="accordion-with-descriptions">\
-        <div class="subsection-wrapper">\
-          <div class="subsection">\
-            <div class="subsection__header">\
-              <h2 class="subsection__title">Subsection title in here</h2>\
-              <p class="subsection__description">Subsection description in here</p>\
-            </div>\
-            <div class="subsection__content" id="subsection_content_0">\
-              <ul class="subsection__list">\
-                <li class="subsection__list-item">\
-                  <a href="">Subsection list item in here</a>\
-                </li>\
-              </ul>\
-            </div>\
+  var html = '\
+    <div class="subsections js-hidden" data-module="accordion-with-descriptions">\
+      <div class="subsection-wrapper">\
+        <div class="subsection">\
+          <div class="subsection__header">\
+            <h2 class="subsection__title" id="topic-section-one">Topic Section One</h2>\
+            <p class="subsection__description">Subsection description in here</p>\
           </div>\
-          <div class="subsection">\
-            <div class="subsection__header">\
-              <h2 class="subsection__title">Subsection title in here</h2>\
-              <p class="subsection__description">Subsection description in here</p>\
-            </div>\
-            <div class="subsection__content" id="subsection_content_1">\
-              <ul class="subsection__list">\
-                <li class="subsection__list-item">\
-                  <a href="">Subsection list item in here</a>\
-                </li>\
-              </ul>\
-            </div>\
+          <div class="subsection__content" id="subsection_content_0">\
+            <ul class="subsection__list">\
+              <li class="subsection__list-item">\
+                <a href="">Subsection list item in here</a>\
+              </li>\
+            </ul>\
           </div>\
         </div>\
-      </div>');
-
-    var accordion = new GOVUK.Modules.AccordionWithDescriptions();
-    accordion.start($element);
-  });
+        <div class="subsection">\
+          <div class="subsection__header">\
+            <h2 class="subsection__title" id="topic-section-two">Topic Section Two</h2>\
+            <p class="subsection__description">Subsection description in here</p>\
+          </div>\
+          <div class="subsection__content" id="subsection_content_1">\
+            <ul class="subsection__list">\
+              <li class="subsection__list-item">\
+                <a href="">Subsection list item in here</a>\
+              </li>\
+            </ul>\
+          </div>\
+        </div>\
+      </div>\
+    </div>';
 
   afterEach(function() {
     $(document).off();
   });
 
-  // Setup
+  describe('when no sub section is linked to', function () {
```

Comment:
Ah, OK. You know my preference is always to have the least amount of stuff happening outside of the example as possible. Just moving that `accordion.start($element);` inside each test would do it.